src/main/java/io/github/oliviercailloux/j_voting/preferences/interfaces/MutablePreference.java

```diff
@@ -0,0 +1,22 @@
+package io.github.oliviercailloux.j_voting.preferences.interfaces;
+
+import com.google.common.graph.MutableGraph;
+
+import io.github.oliviercailloux.j_voting.Alternative;
+
+/**
+ * A mutable preference is a preference in which some alternatives can be added,
+ * which means some edges (pair of alternatives) can be added to the
```

Comment:
in which edges (pairs of alternatives) and alternatives can be added. (Retirer to the corresponding graph). Retirer br, remplacer par paragraphes

Reply:
Retirer to the corresponding graph

FORMAT (et les § doivent rester !)